### ISSUE TITLE
Elastic-agent, Fleet-server (DOCKER) log indexing issue

### ISSUE THREAD

**Issue by andreboscarriol** [elastic/elastic-agent#2360]
Hello!
I'm trying to dash my access and error nginx logs.
It doesn't generate anything for me, but I use beat to get the docker, it gets it normally, apparently the problem is in the indexing.
Please post all questions and issues concerning the Elastic Agent on https://discuss.elastic.co/c/beats
before opening a Github Issue. Your questions will reach a wider audience there,
and if we confirm that there is a bug, then you can open a new issue.

Thats my docker-dashboard:
![image](https://user-images.githubusercontent.com/104112215/224147708-8cbdef6f-eaeb-451c-820c-b8f4ac9df355.png)

Thats my fleet:
![image](https://user-images.githubusercontent.com/104112215/224148662-c3739654-697a-4889-b74f-919e3cb9343d.png)

![image](https://user-images.githubusercontent.com/104112215/224148877-00503f3d-c776-4350-b92c-29f2b6daf321.png)

Thats my elastic-agent:
![image](https://user-images.githubusercontent.com/104112215/224149307-d94ab8b6-f010-4a91-8745-62a6115dd2a8.png)

Thats my elastic-agent logs:
![image](https://user-images.githubusercontent.com/104112215/224149807-7f2ff93f-2fae-4451-b02d-747a43328eab.png)

I did this on another server (Ubuntu 20) and it worked, but the NGINX log paths were different.

For confirmed bugs, please report:
- Version: Linux
- Operating System: CentOs (Fleet in a DOCKER)
- Discuss Forum URL:
- Steps to Reproduce:

**Comment by cmacknz** [comment]
The logs show ApiKey authentication errors, likely it has not been enrolled into Fleet correctly or there is some other setup issue. This could be a bug, but some investigation is required before we can determine that. Please start a discussion on https://discuss.elastic.co/c/elastic-stack/elastic-agent/ and re-open this if a bug has been confirmed.
